auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -107,19 +107,21 @@
           @update:viewOnlyMode="viewOnlyMode"
         />
 
-        <template v-if="baseAddress" v-can:VIEW_ADDRESS.hide>
```

Comment:
I'm guessing Vite didn't like this format? 😆

Reply:
useless directive on template

Reply:
👍